Blogging in html.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>

```

Comment:
Please make the file name all lowercase and following the pattern in the assignment statement